### PR TITLE
Fixes and Addressing Coverage Issues - 1

### PR DESCRIPTION
**Pull request by NobleMathews**
- Fix local clone invocation issues 
- Adds CLI test
- Breaks parser and resolver tests into separate files
-  Issue with .nuspec being parsed incorrectly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,12 @@ A dependency analysis tool for various languages and package managers.
 - Run `pip install -r requirements-dev.txt` to do an editable install
 - Run `pytest` to run tests
 
+```bash
+pytest-watch --runner='coverage run --rcfile=.coveragerc.local -m pytest --testmon' \
+  --onfail="notify-send --urgency=low -i error 'Tests Failed'" \
+  --onpass="coverage html --skip-covered"
+```
+terminal-notifier -message
```
Comment (tushar-deepsource) [inline README.md:16]: What's this?

File: setup.cfg [hunk 2]
```diff
@@ -61,6 +61,8 @@ dev =
     mypy
     pytest
     pytest-cov
+    pytest-watch
```
Comment (tushar-deepsource) [inline setup.cfg:64]: what's this for?

Comment (tushar-deepsource) [reply setup.cfg:64]: Add it in the PR description if it's a new dependency

File: src/depend/cli.py [hunk 3]
```diff
@@ -50,19 +54,19 @@ def main(
...
-        )
-    elif packages:
-        depend = Depend(lang, packages, depth, debug, disable_token)
-    else:
-        logger.error("Nothing to process please specify either dep_file or packages")
-        sys.exit(-1)
     try:
-        result = depend.resolve()
-    except (LanguageNotSupportedError, VCSNotSupportedError, ParamMissing) as e:
+        result = {}
```
Comment (tushar-deepsource) [inline src/depend/cli.py:58]:
```suggestion
```

**Review by tushar-deepsource — CHANGES_REQUESTED** [review]

**Comment by tushar-deepsource** [comment]
Why was `PYPI_LIC` commented out?

File: README.md (hunk 1 above)
Comment (NobleMathews) [reply README.md:16]: Oh it's a command I was using to rerun the tests and coverage report locally whenever I was editing.. should I keep it in dev or just retain it locally

**Comment by NobleMathews** [comment]
> Why was `PYPI_LIC` commented out?

We are not handling the data in it right now and it also comes as missing lines in the cov-report, kept it so we can return to it later

**Comment by tushar-deepsource** [comment]
When will `PYPI_LIC` be used again?

File: README.md (hunk 1 above)
Comment (tushar-deepsource) [reply README.md:16]: The `terminal-notifier -message` thing. That is outside the code block.

Comment (tushar-deepsource) [reply README.md:16]: And add a line explaining that snippet, explaining why you'd want to use it?

Comment (NobleMathews) [reply README.md:16]: Ok will do.. terminal-notifier is to get it to work on mac

**Comment by NobleMathews** [comment]
Whenever we take up processing the license info returned so that we have unique maps to particular licenses no matter the source

File: src/depend/constants.py [hunk 4]
```diff
@@ -194,88 +194,88 @@
...
+#     "OSI Approved :: University of Illinois/NCSA Open Source License",
+#     "OSI Approved :: Vovida Software License 1.0",
+#     "OSI Approved :: W3C License",
+#     "OSI Approved :: X.Net License",
+#     "OSI Approved :: Zope Public License",
+#     "OSI Approved :: zlib/libpng License",
+#     "Other/Proprietary License",
+#     "Public Domain",
+#     "Repoze Public License",
+# ]
```
Comment (tushar-deepsource) [inline src/depend/constants.py:281]:
```suggestion
```

File: README.md (hunk 1 above)
Comment (tushar-deepsource) [reply README.md:16]: Put that line in the code block then?

**Review by tushar-deepsource — APPROVED** [review]

**Pull request merged**